autoload/context.vim

```diff
@@ -276,6 +276,7 @@ function! s:open_preview() abort
                 \ ' buftype=nofile' .
                 \ ' modifiable'     .
                 \ ' nobuflisted'    .
+                \ ' nocursorline'    .
```

Comment:
Could you fix the alignment of the dots? 😉 
```suggestion
                \ ' nocursorline'   .
```

Reply:
Oops. I've fixed it.